tests/tacacs/utils.py

```diff
@@ -338,3 +376,20 @@ def log_exist(duthost):
 
     exist = wait_until(timeout, 1, 0, log_exist, duthost)
     pytest_assert(exist, "Not found aaa config update log: {}".format(command))
+
+
+def print_tacacs_creds(tacacs_creds):
+    print_tacacs_creds = copy.deepcopy(tacacs_creds)
```

Comment:
>print_tacacs_creds [](http://example.com/codeflow?start=4&length=18)

It is weird to the variable name is the same as function name.

Reply:
Fixed，this method removed.